CMakeLists.txt

```diff
@@ -301,6 +303,10 @@ if(openPMD_HAVE_ADIOS1)
     target_link_libraries(openPMD PUBLIC ${ADIOS_LIBRARIES})
     target_include_directories(openPMD SYSTEM PUBLIC ${ADIOS_INCLUDE_DIRS})
     target_compile_definitions(openPMD PUBLIC "-DopenPMD_HAVE_ADIOS1=1")
+    if(ADIOS_HAVE_SEQUENTIAL)
+        # TODO might be smarter to get ALL definitions from adios-config -s & parse it
```

Comment:
Yes, ideally in `FindADIOS.cmake` we should populate `ADIOS_DEFINITIONS`. We then use that var here.